pkgs/development/libraries/vulkan-loader/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv, fetchFromGitHub, cmake, python3, vulkan-headers, pkgconfig,
   xlibsWrapper, libxcb, libXrandr, libXext, wayland, libGL_driver }:
-let version = "1.1.82.0"; in
+let version = "1.1.85"; in
```

Comment:
This breaks this package, because there is no `sdk-1.1.85` branch in the upstream repo. The breakage is silent because you didn't touch the hash so Nix didn't try to fetch a new copy.

Reply:
Oh, good catch. Thank you!

Reply:
Hm, tag is indeed not there, but branch is: https://github.com/KhronosGroup/Vulkan-Loader/tree/sdk-1.1.85

Reply:
Branches are mutable, so you should replace the `rev = ...` line with a reference to the specific git commit currently at the head of the branch, at least.

Reply:
See #48329.